admin/test_langs.php

```diff
@@ -25,7 +25,7 @@
 define( 'PLUGINS_DISABLED', true );
 define( 'LANG_LOAD_DISABLED', true );
 
-$t_mantis_dir = dirname( __DIR__ ) . '/';
+$t_mantis_dir = dirname( __DIR__ ) . DIRECTORY_SEPARATOR;
```

Comment:
Just for the record, using DIRECTORY_SEPARATOR is not needed in this context, PHP will take care of converting `/` as appropriate.